mxresolv/mxresolv.go

```diff
@@ -0,0 +1,145 @@
+package mxresolv
+
+import (
+	"context"
+	"math/rand"
+	"net"
+	"sort"
+	"strings"
+	"unicode"
+
+	"github.com/mailgun/holster/v4/clock"
+	"github.com/mailgun/holster/v4/collections"
+	"github.com/pkg/errors"
```

Comment:
You must use `github.com/mailgun/holster/v4/errors`.
`github.com/pkg/errors` is deprecated.